rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerImplTest.java

```diff
@@ -249,15 +252,118 @@ void logCommitFederation() {
         String oldFederationBtcAddress = oldFederation.getAddress().toBase58();
         byte[] newFederationFlatPubKeys = flatKeysAsByteArray(newFederation.getBtcPublicKeys());
         String newFederationBtcAddress = newFederation.getAddress().toBase58();
-        long newFedActivationBlockNumber = executionBlock.getNumber() + CONSTANTS.getFederationActivationAge();
+        long newFedActivationBlockNumber = executionBlock.getNumber() + CONSTANTS.getFederationActivationAge(activations);
+
+        Object[] data = new Object[]{
+            oldFederationFlatPubKeys,
+            oldFederationBtcAddress,
+            newFederationFlatPubKeys,
+            newFederationBtcAddress,
+            newFedActivationBlockNumber
+        };
+
+        CallTransaction.Function event = BridgeEvents.COMMIT_FEDERATION.getEvent();
+        Object[] topics = {};
+        assertEvent(eventLogs, 0, event, topics, data);
+
+        final LogInfo log = eventLogs.get(0);
+        Object[] decodeEventData = event.decodeEventData(log.getData());
+        long loggedFedActivationBlockNumber = ((BigInteger) decodeEventData[4]).longValue();
+
+        // assert fed activation has different values before and after RSKIP383 respectively
+        ActivationConfig.ForBlock activationsAfterRSKIP383 = mock(ActivationConfig.ForBlock.class);
+        when(activationsAfterRSKIP383.isActive(ConsensusRule.RSKIP383)).thenReturn(true);
+
+        long hopNewFedActivationBlockNumber = executionBlock.getNumber() + CONSTANTS.getFederationActivationAge(activationsAfterRSKIP383);
+        assertNotEquals(loggedFedActivationBlockNumber, hopNewFedActivationBlockNumber);
+        assertNotEquals(newFedActivationBlockNumber, hopNewFedActivationBlockNumber);
+
+        // assert fed activation value for RSKIP383 is being properly logged
+        assertEquals(loggedFedActivationBlockNumber, newFedActivationBlockNumber);
```

Comment:
Isn't this covered in logCommitFederation_after_RSKIP383?

Reply:
Done. Redundant tests were removed.